packages/jetpack-affiliate/lib/Jetpack_Affiliate.php

```diff
@@ -1,25 +1,33 @@
 <?php
+namespace Jetpack\V7\Partners;
 
-if ( ! defined( 'ABSPATH' ) ) {
-	// Exit if accessed directly
-	exit;
+use \Jetpack as Jetpack;
+
+if ( ! defined( 'ABSPATH' ) || ! is_admin() ) {
+	exit; // Exit if accessed directly or not in admin.
 }
 
 /**
  * This class introduces routines to get an affiliate code, that might be obtained from:
- * - an `jetpack_affiliate_code` option in the WP database
+ * - a `jetpack_affiliate_code` option in the WP database
  * - an affiliate code returned by a filter bound to the `jetpack_affiliate_code` filter hook
  *
  * @since 6.9.0
  */
 class Jetpack_Affiliate {
```

Comment:
Since we are namespacing things why not just Affiliate? as the class Name?

Reply:
True, fixed